Nos/Extensions/URL+Extensions.swift

```diff
@@ -43,4 +43,8 @@ extension URL {
             return "[\(host)...](\(url.absoluteString))"
         }
     }
+    
+    var isValidNoteLink: Bool {
```

Comment:
The computed var is almost self-explanatory, but I think there is still a benefit if we document it anyway so that new developers can understand what they are. Maybe a documentation entry explaining how note links look like and that they are generated by NoteParser would do.

Reply:
Yeah, good point. However, I ended up not using this in the end, so I'll just remove it for now.